package.json

```diff
@@ -74,8 +74,8 @@
         "jest-axe": "^5.0.1",
         "jest-sonar-reporter": "^2.0.0",
         "prettier": "^2.3.1",
-        "react": "^17.0.2",
-        "react-dom": "^17.0.2",
+        "react": "^17.x",
+        "react-dom": "^17.x",
```

Comment:
React shouldn't be in both dependencies and peer dependencies, should it?

Reply:
Has to be in devDependencies so that it is installed when we compile and test because npm 6 does not auto install peerDependencies.

The published package does not include devDependencies, but anyone using the library will be notified because of the peerDependency.  

npm 7 would make it so we didnt need the devDependencies, but for npm 6 we do

Reply:
Ah, thanks. I didn't notice this was the devDependencies section.